src/middlewared/middlewared/plugins/system.py

```diff
@@ -75,6 +75,11 @@ async def system_advanced_extend(self, data):
         if data.get('sed_user'):
             data['sed_user'] = data.get('sed_user').upper()
 
+        if data['swapondrive'] and (
+            not await self.middleware.call('system.is_freenas') and await self.middleware.call('failover.licensed')
+        ):
```

Comment:
Nothing related to failover licensed, it should be for all TrueNAS, if 12.0 and 12.1 check for failover licensed we need to remove that.